src/definitions/mso.rs

```diff
@@ -9,14 +42,26 @@ use std::collections::BTreeMap;
 pub struct DigestId(i32);
 pub type DigestIds = BTreeMap<DigestId, ByteStr>;
 
+/// Represents an [Mso] object.
```

Comment:
We should probably expand the abbreviation to `Master Signing Object` for this object.

```rust
/// Represents a Master Signing Object (MSO)
```

I saw `MSO (Mobile Security Officer)` above and was confused for a moment. We should try to expand abbreviations where possible.

Reply:
Thoughts on changing this struct name to `MasterSigningObject` ? I know its more verbose, but open to thoughts.

Reply:
MSO has a specific meaning from 18013-5: mobile security object